### PR TITLE
exposes loglevel of connect components

### PR DESCRIPTION
**Pull request by akohlmann**
Exposes env vars to set loglevel for api and sync components

### CODE REVIEW

**Comment by hpatel826** [comment]
I think this is a good solution for https://github.com/1Password/connect/issues/44 instead of changing the connect application's logging levels. Our company also uses Datadog where some systems conform to the syslog `level` field, but 1Password does not.

**Comment by pesoos** [comment]
@jpcoenen please review

File: charts/connect/README.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ $ helm install --set connect.applicationName=connect connect ./connect
 | connect.api.resources | object | `{}` | The resources requests/limits for the 1Password Connect API pod |
 | connect.api.httpPort | integer | `8080` | The port the Connect API is served on when TLS is disabled |
 | connect.api.httpsPort | integer | `8443` | The port the Connect API is served on when TLS is enabled |
+| connect.api.loglevel | string | `info` | Log level of the Connect API container |
```
Comment (jpcoenen) [inline charts/connect/README.md:56]: What do you think about listing all the valid options? 
```suggestion
| connect.api.logLevel | string | `info` | Log level of the Connect API container. Valid options are: trace, debug, info, warn, error. |
```

File: charts/connect/README.md [hunk 2]
```diff
@@ -82,6 +83,7 @@ $ helm install --set connect.applicationName=connect connect ./connect
 | connect.sync.name | string | `"connect-sync"` | The name of the 1Password Connect Sync container |
 | connect.sync.resources | object | `{}` | The resources requests/limits for the 1Password Connect Sync pod |
 | connect.sync.httpPort | integer | `8081` | The port serving the health of the Sync container |
+| connect.sync.loglevel | string | `info` | Log level of the Connect Sync container |
```
Comment (jpcoenen) [inline charts/connect/README.md:86]: Same suggestions as above
```suggestion
| connect.sync.logLevel | string | `info` | Log level of the Connect Sync container. Valid options are: trace, debug, info, warn, error. |
```

File: charts/connect/templates/connect-deployment.yaml [hunk 3]
```diff
@@ -80,6 +80,8 @@ spec:
             - name: OP_HTTP_PORT
               value: "{{ .Values.connect.api.httpPort }}"
             {{- end }}
+            - name: OP_LOG_LEVEL
+              value: "{{ .Values.connect.api.loglevel }}"
```
Comment (jpcoenen) [inline charts/connect/templates/connect-deployment.yaml:84]:
```suggestion
              value: "{{ .Values.connect.api.logLevel }}"
```

File: charts/connect/templates/connect-deployment.yaml [hunk 4]
```diff
@@ -126,6 +128,8 @@ spec:
               value: "11221"
             - name: OP_BUS_PEERS
               value: localhost:11220
+            - name: OP_LOG_LEVEL
+              value: "{{ .Values.connect.sync.loglevel }}"
```
Comment (jpcoenen) [inline charts/connect/templates/connect-deployment.yaml:132]:
```suggestion
              value: "{{ .Values.connect.sync.logLevel }}"
```

File: charts/connect/values.yaml [hunk 5]
```diff
@@ -15,13 +15,15 @@ connect:
     resources: {}
     httpPort: 8080
     httpsPort: 8443
+    loglevel: info
```
Comment (jpcoenen) [inline charts/connect/values.yaml:18]:
```suggestion
    logLevel: info
```

**Review by jpcoenen — COMMENTED** [review]
Hey! Thanks for this contribution. Simple but definitely useful.

What do you think about using `logLevel` instead of `loglevel`? I think it's to a bit more in line with the other values. If you agree, you can just accept all my change suggestions :)

**Comment by akohlmann** [comment]
> Hey! Thanks for this contribution. Simple but definitely useful.
> 
> What do you think about using `logLevel` instead of `loglevel`? I think it's to a bit more in line with the other values. If you agree, you can just accept all my change suggestions :)

@jpcoenen totally reasonable, thanks for the improvements 🙏

**Review by jpcoenen — APPROVED** [review]
I've just triggered the pipelines. Let's merge this after they have run 🚀

**Comment by jpcoenen** [comment]
I just wanted to merge this, but I see this PR has some unsigned commits.

To get your PR merged, please sign all commits (`git commit -S --amend --no-edit`) and then force push them to this branch (`git push --force-with-lease`). Apologies for the inconvenience!

If you're new to commit signing, there are different ways to set it up:

<details>
<summary><b>Sign commits with <code>gpg</code></b></summary>
<p></p>

Follow the steps below to set up commit signing with `gpg`:
1. [Generate a GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key)
2. [Add the GPG key to your GitHub account](https://docs.github.com/en/authentication/managing-commit-signature-verification/adding-a-gpg-key-to-your-github-account)
3. [Configure `git` to use your GPG key for commits signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-gpg-key)

</details>

<details>
<summary><b>Sign commits with <code>ssh-agent</code></b></summary>
<p></p>

Follow the steps below to set up commit signing with `ssh-agent`:
1. [Generate an SSH key and add it to `ssh-agent`](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
2. [Add the SSH key to your GitHub account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)
3. [Configure `git` to use your SSH key for commits signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key)

</details>

<details>
<summary><b>Sign commits with 1Password</b></summary>
<p></p>

You can also sign commits using [1Password](https://developer.1password.com/docs/ssh/git-commit-signing/):

<a href="https://www.youtube.com/watch?feature=player_embedded&v=BMFvhl0WRFQ" target="_blank">
  Watch the demo
</a>

</details>

**Comment by akohlmann** [comment]
> I just wanted to merge this, but I see this PR has some unsigned commits.
> 
> To get your PR merged, please sign all commits (`git commit -S --amend --no-edit`) and then force push them to this branch (`git push --force-with-lease`). Apologies for the inconvenience!
> 
> If you're new to commit signing, there are different ways to set it up:
> 
> Sign commits with `gpg`
> Sign commits with `ssh-agent`
> Sign commits with 1Password

@jpcoenen done, also thanks for the 1Password commit signing tip, awesome feature 🚀

**Pull request merged**